src/robot_state_publisher.cpp

```diff
@@ -58,7 +62,7 @@ void RobotStatePublisher::addChildren(const KDL::SegmentMap::const_iterator segm
   const std::string& root = GetTreeElementSegment(segment->second).getName();
 
   const std::vector<KDL::SegmentMap::const_iterator>& children = GetTreeElementChildren(segment->second);
-  for (unsigned int i=0; i<children.size(); i++) {
+  for (unsigned int i = 0; i < children.size(); ++i) {
```

Comment:
`size_t` for i.

Reply:
size_t in d780506c444f1519c78c29717b7f4296a884b016